src/main/java/com/sparta/gourmate/domain/order/controller/OrderController.java

```diff
@@ -0,0 +1,80 @@
+package com.sparta.gourmate.domain.order.controller;
+
+import com.sparta.gourmate.domain.order.dto.OrderRequestDto;
+import com.sparta.gourmate.domain.order.entity.Order;
+import com.sparta.gourmate.domain.order.service.OrderService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/api/orders")
+public class OrderController {
+
+    private final OrderService orderService;
+
+    public OrderController(OrderService orderService) {
+        this.orderService = orderService;
+    }
+
+    // 주문 생성 (POST)
+    @PostMapping
+    public ResponseEntity<Order> createOrder(@RequestBody OrderRequestDto orderRequestDto) {
```

Comment:
RequestDto에 Validation이 필요해요
```suggestion
    public ResponseEntity<Order> createOrder(@Valid @RequestBody OrderRequestDto orderRequestDto) {
```